tests/mvp_test.py

```diff
@@ -0,0 +1,307 @@
+import copy
```

Comment:
It would be nice to have some comments at the top of this file saying a bit about what it tests.

Reply:
Done. The example from EECI is actually in this file (mvp_test() function). I think it's better to put it as part of tests instead of examples so that we keep checking that it always works.